spelling/styles/Consensys/Punctuation.yml

```diff
@@ -2,8 +2,12 @@ extends: substitution
 message: "Prefer Consensys style."
 level: error
 ignorecase: true
+# This is not working
 action:
   name: replace
 swap:
-  ′: "'"
-  ″: '"'
\ No newline at end of file
+  '′': "'"
```

Comment:
@joshuafernandes I have tried various approaches to handle this a per @alexandratran request. Cannie do it. Can't find anything to suggest that Vale should be able to do this --> should be something the code linter does rather?

Reply:
I'll add this to my list and come back to this one @m4sterbunny